pkg/glusterfs/controllerserver.go

```diff
@@ -183,56 +148,70 @@ func (cs *ControllerServer) validateCreateVolumeReq(req *csi.CreateVolumeRequest
 	return nil
 }
 
-func (cs *ControllerServer) checkExistingVolume(volumeName string, volSizeMB int) (string, []string, error) {
-	var (
-		tspServers  []string
-		mountServer string
-		err         error
-	)
+func (cs *ControllerServer) doVolumeCreate(volumeName string, volSizeMB int) error {
+	glog.V(4).Infof("Received request to create volume %s with size %d", volumeName, volSizeMB)
+	volMetaMap := make(map[string]string)
+	volMetaMap[volumeOwnerAnn] = glusterfsCSIDriverName
+	volumeReq := api.VolCreateReq{
+		Name:         volumeName,
+		Metadata:     volMetaMap,
+		ReplicaCount: defaultReplicaCount,
+		Size:         uint64(volSizeMB),
+	}
+
+	glog.V(2).Infof("volume create request: %+v", volumeReq)
```

Comment:
if we have a debug log option this can be added as a debug log

Reply:
sorry this can be removed as we are logging volume create request in `volumeCreate` function

Reply:
@jarrpa this is pending

Reply:
We log the CSI CreateVolume request earlier, but we don't log the GD2 VolumeCreate request. :)